Index: src/uthash.h
===================================================================
--- src/uthash.h	(revision 123)
+++ src/uthash.h	(working copy)
@@ -26,7 +26,8 @@
 
 #include <string.h>   /* memcmp,strlen */
 #include <stddef.h>   /* ptrdiff_t */
-#include <inttypes.h> /* uint32_t etc */
+//#include <inttypes.h> /* uint32_t etc */
+typedef unsigned int uint32_t;
 
 #define UTHASH_VERSION 1.8
 
